poll/tests.py

```diff
@@ -1,3 +1,75 @@
-# from django.test import TestCase
+import pytest
+from .models import Poll, get_default_end_date
+from datetime import datetime, timedelta
+from event.models import Event
+from category.models import Category
+from location.models import Location
+from category_location.models import CategoryLocation
 
-# Create your tests here.
+EVENT_NAME = "weight lifting"
+CATEGORY = "Gym"
+LOCATION = "Holmes place Netanya"
+MAX_PART = 10
+IS_PRIVATE = False
+DATETIME = datetime.now()
+
+
+@pytest.mark.django_db()
+def create_sample_category_location(
+    cat_name="test",
+    loc_name="test",
+    loc_city="test",
+    loc_street="test",
+    loc_street_num=1,
+    loc_indoor=False,
+    loc_des="test",
+):
+    loc = Location(
+        name=loc_name,
+        city=loc_city,
+        street=loc_street,
+        street_number=loc_street_num,
+        indoor=loc_indoor,
+        description=loc_des,
+    )
```

Comment:
also change it like:
```suggestion
def category_location1()
    loc = Location(
        name=LOCATION,
        city="test city",
        street="test street",
        street_number=1,
        indoor=False,
        description="test description",
    )
```
Now you can remove lines # 43-45